src/agenda/index.js

```diff
@@ -55,6 +55,8 @@ export default class AgendaView extends Component {
     renderEmptyData: PropTypes.func,
     // specify your item comparison function for increased performance
     rowHasChanged: PropTypes.func,
+    
+    shouldChangeDay: PropTypes.func,
```

Comment:
What is this prop for?

Reply:
The default value of the 'shouldChangeDay' prop should be true, otherwise, this is a breaking change